app/src/main/java/org/oppia/app/customview/inputInteractionView/FractionInputInteractionView.kt

```diff
@@ -0,0 +1,42 @@
+package org.oppia.app.customview.inputInteractionView
+
+import android.content.Context
+import android.text.InputFilter
+import android.text.method.DigitsKeyListener
+import android.widget.EditText
+import org.oppia.app.R
+
+/** The customclass for [FractionInputInteractionView]. */
+class FractionInputInteractionView : EditText {
+
+  private var mContext: Context
+  private var placeholder: String
+  private var rows: Int
+
+  constructor(context: Context, placeholder: String, rows: Int) : super(context) {
+    this.mContext = context
+    this.placeholder = placeholder
+    this.rows = rows
+    init()
+  }
+
+  //update default attributes of TextInputInteractionView here
```

Comment:
Write comments properly.
Example: `// This function contains default attributes of TextInputInteractionView. `

Reply:
Nit: After `//` there should be one space, first character should be capital and before end of sentence there should be `.`

Reply:
okay